gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -7,28 +7,31 @@ module ReactRenderable
     include ReverseProxy::Controller
 
     def render_react
+      if defined? ShopifySecurityBase
+        ShopifySecurityBase::HTTPHostRestriction.whitelist([Quilt.configuration.react_server_host]) do
```

Comment:
You could redo this as `ShopifySecurityBase::HTTPHostRestriction.whitelist([Quilt.configuration.react_server_host], &method(:proxy))` (I think)

Reply:
personally I find this slightly more readable, but that might just be due to my dislike of/tendency to be confused by the unary ampersand operator :P